src/aws/mod.rs

```diff
@@ -43,10 +43,19 @@ pub fn is_retriable_error<T>(error: &SdkError<T>) -> bool {
     match error {
         SdkError::TimeoutError(_) | SdkError::DispatchFailure(_) => true,
         SdkError::ConstructionFailure(_) => false,
-        SdkError::ResponseError(err) => check_response(err.raw()),
-        SdkError::ServiceError(err) => check_response(err.raw()),
+        SdkError::ResponseError(err) => {
+            let raw = err.raw();
+
+            check_response(raw)
+        }
+        SdkError::ServiceError(err) => {
+            let raw = err.raw();
+
+            check_response(raw)
+        }
```

Comment:
```suggestion
        SdkError::ResponseError(err) => {
            check_response(err.raw())
        }
        SdkError::ServiceError(err) => {
            check_response(err.raw())
        }
```

Another clean up from the PR review before